src/main/pg/rx.c

```diff
@@ -114,6 +115,10 @@ void pgResetFn_rxConfig(rxConfig_t *rxConfig)
         .sbus_baud_fast = false,
         .msp_override_channels_mask = 0,
         .crsf_use_negotiated_baud = false,
+        .headtracker_max_angle = 0,
```

Comment:
headtracker should use separate PG. It is not related to RX

Reply:
Done